doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,230 @@
+Thanks to the descriptor file, the NFVi infrastructure deployment could be completed in one step run.
 
 
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.gen.ins.01` | Installer | Installer **should** accept a descriptor file to finish deployment.|
+| `req.gen.ins.02` | Installer | Installer implementation **must** validate the descriptor file with schema.|
+| `req.gen.ins.03` | Installer | Any existing installer implementation **may** need adaption for the descriptor file. |
+| `req.gen.ins.04` | Installer | Installer **may** support reporting the deplolyment progress status.|
+| `req.gen.des.01` | Descriptor | Descriptor file **should** include hardware resource configuration, software configuraiton.|
+| `req.gen.des.02` | Descriptor | Descriptor file **may** include additional extending configuraiton.|
+
+<a name="5.2.2"></a>
+### 5.2.2 Additional
+Depends xxx.
+
 <a name="5.3"></a>
-## 5.3 Heading
+## 5.3 Descriptor definition
+There should be a Descriptor File definition, which used by installer as input of necessary configuration.
+Mandatory and optional definition should be defined.
+ 
+<a name="5.3.1"></a>
+### 5.3.1 Hardware resource information
+The support of different workload types, each with different compute, storage, network requirements which needs kinds of hardware configuration template, we name this as server template.
+Besides it may include optional information such area name, data center name etc.
+
+| Field # | type | mandatory? | Instruction |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| area_name | String | Yes | data center area name |
+| area_center_name | String | Yes | data center name, may compliance with a naming rule.  |
+| room_name | String | Yes | rome static data, helpful in locating issue occurred. |
+| city | String | Yes |  |
+| resourece_pool_name | String | Yes |  |
+| outband_network_segment | String | Yes |  |
+| server_templates | List | Yes |  |
+| server_infos | List | Yes |  |
+
+<p align="center"><b>Table 5-3-1:</b> Hardware resource description.</p>
+
+### 5.3.2 Server template
+The server template describes the capability of the host like processor, memory, harddrive, raid, NIC/NIC binding, manufacturer, model etc.
+Server template would be assigned to multiple servers, i.e phyiscal hosts.
+| Field # | type | mandatory? | Instruction |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Shall we also add iLO or iDRAC IP info here ?

Reply:
I think you are right, some attributes about the remote management should be added. (like ILO ip, user/password). Just thinking where should I put those attributes?  It would be better to add on server template if all of them configured default and same value. otherwise, they belong to server.